core/src/test-core/java/com/radixdlt/environment/deterministic/MultiNodeDeterministicRunner.java

```diff
@@ -104,7 +105,7 @@ public Injector getNode(int index) {
     return nodes.get(index);
   }
 
-  public <T> void dispatchToAll(Key<EventDispatcher<T>> dispatcherKey, T t) {
+  public <T extends LocalEvent> void dispatchToAll(Key<EventDispatcher<T>> dispatcherKey, T t) {
```

Comment:
looks like we can remove this method; it's no longer used

Reply:
Done